pkg/cmd/server/api/types.go

```diff
@@ -6,6 +6,14 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
 )
 
+type OpenShiftFeature int
```

Comment:
Well the integer values would get converted to strings during serialization. But I guess it adds unnecessary complexity indeed.